node/config/def.go

```diff
@@ -117,6 +117,45 @@ func DefaultFullNode() *FullNode {
 	}
 }
 
+// DefaultFullNode returns the default config
+func DefaultFollowerNode() *Follower {
```

Comment:
Is this used? Follower node is still a "FullNode" node type, it shouldn't be a different type.

(FullNode should really be called ChainNode, but renaming that now is not really relevant to this PR)

Reply:
not used, thanks, deleted it aa059e64c